CMSgov/easi-app
# [Easi-3109] Email notifications for close/reopen system intake

# EASI-3109

## Changes and Description

- Adds notification emails to the following Admin Actions:
    - Close Request
    - Re-open Request

## How to test this change

Use postman to do the following ( or use the front end)

1. Create a system intake and submit it.
1. Close the system intake
1. Confirm that you receive an email in mail catcher
1. Re-open the system intake
1. Confirm that you receive an email in mail catcher
1. Send email that does not include `additionalInfo`.
1. Confirm that the additional information section is left out of the email

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
